internal/engine/probeservices/measurementmeta_test.go

```diff
@@ -52,7 +52,7 @@ func TestGetMeasurementMetaWorkingAsIntended(t *testing.T) {
 	if mmeta.Input == nil || *mmeta.Input != config.Input {
 		t.Fatal("unexpected input value")
 	}
-	if mmeta.MeasurementStartTime.String() != "2020-12-09 05:22:25 +0000 UTC" {
+	if mmeta.MeasurementStartTime.String() != "2020-12-09 04:22:25 +0000 UTC" {
```

Comment:
I don't fully understand why this change is necessary and the commit does not seem to explain in detail why this was behaving in a flaky way. Would you mind clarifying?

Reply:
There was a bug in the data pipeline (as I had informed in our 1:1). I have reverted this back since the bug has been fixed now: https://github.com/ooni/probe-cli/pull/846/commits/f71c4e29d5c413254ef6d37884abe2bf0a86bdfe